modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/exec/rel/AbstractExecutionTest.java

```diff
@@ -114,6 +142,59 @@ public void setup() throws Exception {
         }
     }
 
+    /** Task reordering executor. */
+    private static class IgniteRandomStripedThreadPoolExecutor extends IgniteStripedThreadPoolExecutor {
```

Comment:
Naming suggestion: IgniteTestStripedThreadPoolExecutor. Not only random mode is supported